avb/utils/debug.py

```diff
@@ -0,0 +1,4 @@
+import numpy as np
```

Comment:
The path `avb/...` doesn't exist anymore and has been refactored to `playground/...` and in general all these debug files are unnecessary for the master branch. I also have such scratch files for testing but they are ignored or untracked. Please do the same and remove these (or if you don't know how, I can merge it like that and remove it with the next commit to master).